### DISCUSSION TITLE
Welcome to libpqxx Discussions!

### DISCUSSION THREAD

**Discussion by jtv** [jtv/libpqxx#725] [Announcements]
Please help me try out Github's Discussions feature.  Post here for conversations that aren't quite an Issue: finding help, sharing ideas, asking questions about practices, talking over design options for new features.

If it works well, we'll keep it.  :-)

**Comment by pkfox** [comment]
@jtv Should I post here instead of issues ?

**Reply by jtv** [reply]
Feel free to start a fresh discussion.

**Reply by pkfox** [reply]
@jtv Any progress on the to_buf saga

**Reply by jtv** [reply]
I think so, yes.  I'm a bit fuzzy right now but I think some problems were resolved since we last spoke.
